services/gateway/starter/src/main/resources/gateway.conf

```diff
@@ -372,7 +379,7 @@ akka {
         #
         # `relaxed`: all visible 7-Bit ASCII chars are allowed
         #
-        uri-parsing-mode = relaxed
+        uri-parsing-mode = strict
```

Comment:
Is this a breaking change in ditto?

Reply:
hmmh, I think we once configured `strict` for a very good reason. (that I can't think of any more)
why was this changed to `relaxed` again?

Reply:
I think you're right. Let's revert this change and set it to relaxed again.